src/client/home/HomePage.tsx

```diff
@@ -1,18 +1,30 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import MintSection from './MintSection/MintSection';
 import ImageSection from './ImageSection/ImageSection';
+import { useAuth } from '@/client/home/useAuth';
+import Header from './Header/Header';
 
 interface HomePageProps {
   className?: string;
 }
 
 const HomePage: React.FC<HomePageProps> = ({ className }) => {
+  const { login, address} = useAuth();
 
+  useEffect(() =>{
+    if (!address)  {
+      login();
+    }
+  }, [address, login]) 
```

Comment:
I want the login model to open up the second the page renders so that we can collect addresses to track the rest of the actions, anyway around this useEffect @maximeantoine1997 ?

Reply:
this is fine, just put a comment, to explain why we do this